GitUtils.ps1

```diff
@@ -117,45 +117,73 @@ function Get-GitStatus($gitDir = (Get-GitDirectory)) {
         $filesUnmerged = @()
 
         if($settings.EnableFileStatus -and !$(InDisabledRepository)) {
-            dbg 'Getting status' $sw
-            $status = git -c color.status=false status --short --branch 2>$null
-        } else {

```

Comment:
This is what I was thinking we needed, but you're absolutely right that it's dead code.

```powershell
        if($settings.EnableFileStatus -and !$(InDisabledRepository)) {
            if ($settings.EnableFileStatusFromCache -eq $null) {
                $settings.EnableFileStatusFromCache = (Get-Module GitStatusCachePoshClient) -ne $null
            }

            if ($settings.EnableFileStatusFromCache) {
                dbg 'Getting status from cache' $sw
                ...
            } else {
                dbg 'Getting status' $sw
                $status = ...
            }
        } else {
            $status = @()
            # Never used            
        }
```